src/net.h

```diff
@@ -489,9 +489,8 @@ class CNode
     /** Whether this peer provides all services that we want. Used for eviction decisions */
     std::atomic_bool m_has_all_wanted_services{false};
 
-    /** Whether we should relay transactions to this peer (their version
-     *  message did not include fRelay=false and this is not a block-relay-only
-     *  connection). This only changes from false to true. It will never change
+    /** Whether we should relay transactions to this peer.
+     *  This only changes from false to true. It will never change
      *  back to false. Used only in inbound eviction logic. */
```

Comment:
Since this line is touched anyway "Used only in inbound eviction logic" is no longer the case, Erlay also uses it.

Reply:
Thanks, done